src/main/kotlin/no/nav/familie/dokument/storage/RestExceptionHandler.kt

```diff
@@ -0,0 +1,57 @@
+package no.nav.familie.dokument.storage
+
+import org.slf4j.LoggerFactory
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.ControllerAdvice
+import org.springframework.web.bind.annotation.ExceptionHandler
+import org.springframework.web.context.request.ServletWebRequest
+import java.time.LocalDateTime
+import java.util.LinkedHashMap
+
+@ControllerAdvice
+class RestExceptionHandler {
+
+    @ExceptionHandler(IllegalArgumentException::class)
```

Comment:
We really needed an ExceptionHandler, but it have to be a bit more complex. I created one first that takes care of all Runtime and returns 500, that failed hard. So i improved it in https://github.com/navikt/familie-dokument/pull/44 
So it looks almost the same as in familie-ef-soknad-api

It logs and takes handle of `JwtTokenUnauthorizedException` so it returns 401 if it is thrown.